Sources/LocalReceiptParsing/Helpers/ReceiptParserLogger.swift

```diff
@@ -16,87 +16,85 @@ import Foundation
 final class ReceiptParserLogger: LoggerType {
 
     func verbose(
-        _ message: @autoclosure () -> LogMessage,
+        _ message: LogMessage,
         fileName: String?,
         functionName: String?,
         line: UInt
     ) {
         Self.log(
             level: .verbose,
-            message: message(),
+            message: message,
             fileName: fileName,
             functionName: functionName,
             line: line
         )
     }
 
     func debug(
-        _ message: @autoclosure () -> LogMessage,
+        _ message: LogMessage,
         fileName: String?,
         functionName: String?,
         line: UInt
     ) {
         Self.log(
             level: .debug,
-            message: message(),
+            message: message,
             fileName: fileName,
             functionName: functionName,
             line: line
         )
     }
 
     func info(
-        _ message: @autoclosure () -> LogMessage,
+        _ message: LogMessage,
         fileName: String?,
         functionName: String?,
         line: UInt
     ) {
         Self.log(
             level: .info,
-            message: message(),
+            message: message,
             fileName: fileName,
             functionName: functionName,
             line: line
         )
     }
 
     func warn(
-        _ message: @autoclosure () -> LogMessage,
+        _ message: LogMessage,
         fileName: String?,
         functionName: String?,
         line: UInt
     ) {
         Self.log(
             level: .warn,
-            message: message(),
+            message: message,
             fileName: fileName,
             functionName: functionName,
             line: line
         )
     }
 
     func error(
-        _ message: @autoclosure () -> LogMessage,
+        _ message: LogMessage,
         fileName: String,
         functionName: String,
         line: UInt
     ) {
         Self.log(
             level: .error,
-            message: message(),
+            message: message,
             fileName: fileName,
             functionName: functionName,
             line: line
         )
     }
 
     private static func log(level: LogLevel,
-                            message: @autoclosure () -> LogMessage,
-                            fileName: String? = #fileID,
-                            functionName: String? = #function,
-                            line: UInt = #line) {
-        let message = message()
-
+                            message: LogMessage,
+                            fileName: String?,
+                            functionName: String?,
+                            line: UInt) {
```

Comment:
I'm curious. Do we gain something with this? (FWIW, I totally agree with removing these default values since they're never used in this `private` method)